maintainers/maintainer-list.nix

```diff
@@ -2590,6 +2590,12 @@
       }
     ];
   };
+  Ch1keen = {
+    email = "gihoong7@gmail.com";
+    github = "Ch1keen";
+    githubId = 40013212;
+    name = "Han Jeongjun";
+  };
```

Comment:
Please add maintainer in a separate commit.

Reply:
Should I rebase the commit?

Reply:
No, split into two commits, see https://github.com/NixOS/nixpkgs/pull/222982/commits